AudioDeviceExample/AudioDevices/ExampleAVAudioEngineDevice.m

```diff
@@ -336,15 +337,18 @@ - (BOOL)startRendering:(nonnull TVIAudioDeviceContext)context {
                                     captureContext:self.capturingContext]) {
             return NO;
         }
-        return [self startAudioUnit];
+        BOOL success = [self startAudioUnit];
+        if (success) {
+            TVIAudioSessionActivated(context);
+        }
+        return success;
```

Comment:
I think this is the correct place notify sdk about the AVAudioSession activation. Webrtc notifies RTCAudioSessionDelegate in init though because they initialize audio unit in AudioDeviceModule's initPlayout (i.e. AudioDeviceIOS::AVinitPlayOrRecord).

Reply:
This is the approach that we kept. If we decide to change it in the future, we can issue another PR.